js_modules/dagster-ui/packages/ui-core/src/search/SearchResults.tsx

```diff
@@ -48,6 +48,8 @@ const iconForType = (type: SearchResultType | AssetFilterSearchResultType): Icon
       return 'asset_group';
     case AssetFilterSearchResultType.ComputeKind:
       return 'tag';
```

Comment:
why is this also `tag` ?

Reply:
Hm, I think we should update this to use the compute kind icon.

But I can do that in a follow up PR.